README.md

```diff
@@ -47,6 +47,14 @@ docker:
     - "/path/to/another-image-name.tar"
 ```
 
+### Run a Command
+device-init can execute a list of commands on boot. e.g.:
+```yaml
+runcmd:
+  - "apt-get install package-name"
```

Comment:
Interesting. My assumption was that yaml file should have correct syntax, otherwise we might eventually end up with error while decoding the file. I guess it depends on the yaml library used underneath and how it handles such cases.